### PR TITLE
Migrate VSCE to use local-dev-lib instead of cli-lib

### PR DESCRIPTION
**Pull request by jsines**
Hits the couple tasks here: https://github.com/HubSpot/hubspot-cms-vscode/issues/247
Depends on this PR from local-dev-lib: https://github.com/HubSpot/hubspot-local-dev-lib/pull/134

Diff size is mostly `package-lock.json` and file removals 😄

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,5 +1,2 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
-
-node ./scripts/sanityCheckEnLyaml.js
```
Comment (jsines) [inline .husky/pre-commit:4]: A little historical context on these lang changes - with `cli-lib` there was no bundling process and the `lyaml` files didn't get shipped with the package, so I had set it up to download the lang data on commit to keep it updated and then we could bundle it like this. No longer necessary with `local-dev-lib` bundling its own lang data so we don't see those failures

File: src/lib/types.ts [hunk 2]
```diff
@@ -42,18 +42,6 @@ export function instanceOfCommand(object: any): object is Command {
...
-  message: string;
-  lineno: number;
-  startPosition: number;
-  categoryErrors: {
-    fullName?: string;
-    path?: string;
-  };
-  category: string;
-}
-
```
Comment (jsines) [inline src/lib/types.ts:56]: This type exists in `local-dev-lib` now, so changing our usage to that one and removing this. Theres some others I'm sure that we could swap to LDL types but this is just one I specifically noted

File: src/lib/uri.ts [hunk 3]
```diff
@@ -48,12 +49,18 @@ const handleAuthRequest = async (authParams: URLSearchParams) => {
...
+    throw err;
+  }
+  const updatedConfig = await updateConfigWithAccessToken(
+    token,
     personalAccessKey,
-    name,
     env,
-  });
+    name
+  );
```
Comment (jsines) [inline src/lib/uri.ts:63]: `updateConfigWithPersonalAccessKey` no longer exists in LDL so fetching token first and then using `updateConfigWithAccessToken`

**Review by j-malt — APPROVED** [review]
Looks good!

**Review by TanyaScales — APPROVED** [review]
LGTM

**Pull request merged**